.gitignore

```diff
@@ -1,12 +1,12 @@
 *.egg-info/*
 .coverage
 __pycache__
+assets/
 bin/
+contracts/
+docs/_build/
+invoices/
 lib*
 pyvenv.cfg
+settings.yaml
```

Comment:
Checking the current `.gitignore` with the `./` lead-in using `git check-ignore --verbose`, it seems indeed like these rules are broken and do never match.

However, it does not really seem like a good idea to me to add things like `contracts/` or `invoices/` to `.gitignore` without making them relative to the repository root:

Imagine you are changing the `contract.py` module in `rechnung` into its own `contracts` package.

With your change, this whole package would be ignored by git, because any directory that goes by the name of `contracts` will be ignored anywhere in the repository.

The better solution might be to change all occurrences of `./` in the `.gitignore` with `/`:
- It's an allowed pattern
- It seems to match the originally intended meaning
- It doesn't ignore directories with common names broadly and unspecifically